src/index.test.js

```diff
@@ -28,11 +29,23 @@ describe('css-vendor', () => {
       expect(supportedProperty('display')).to.be('display')
     })
 
-    it('should prefix if needed', function () {
-      const {level, needPrefix} = getSupport('transforms2d')
-      if (level === 'none' || level === 'unknown') this.skip()
-      const prop = needPrefix ? `${prefix.css}transform` : 'transform'
-      expect(supportedProperty('transform')).to.be(prop)
+    it('should prefix if needed', () => {
+      const failMsgs = []
+      for (const property in propertyPrefixFixture) {
+        try {
+          expect(supportedProperty(property)).to.be(propertyPrefixFixture[property])
+        }
+        catch (e) {
+          failMsgs.push(`${e} for '${property}'`)
+        }
+      }
+      if (failMsgs.length > 0) {
+        failMsgs.forEach(err => {
+          const logMsg = `${currentBrowser.id} ${currentBrowser.version} ${err}`
+          console.log(logMsg) // eslint-disable-line no-console
```

Comment:
True lets do that. But we will still need to log to the console, because of the karma-mocha-reporter issue. The problem is that when two browsers fail the same test but outputs a different message, only the first message is reported in the summary.